.gitignore

```diff
@@ -1,4 +1,4 @@
-/target
```

Comment:
on my system running `just run` from the root dir still creates a target dir in the root. so maybe we need `/target` and `*/target`?

Reply:
thanks, fixed